static/src/js/pages/SearchPage/SearchPage.jsx

```diff
@@ -134,10 +134,10 @@ const SearchPage = ({ limit }) => {
   }
 
   const buildEllipsisLinkCell = useCallback((cellData, rowData) => {
-    const { conceptId } = rowData
+    const { conceptId, revisionId } = rowData
 
     return (
-      <EllipsisLink to={`/collections/${conceptId}`}>
+      <EllipsisLink to={`/collections/${conceptId}/${revisionId}`}>
```

Comment:
says line 275 (below) is missing a test

Reply:
This was fixed by removing the `.only` from this test file

Reply:
line 275 is a sortFn, there is an test for that. I have verified that making this revisionId change did not cause that line to be uncovered. 

```
{
  dataKey: 'longName',
  title: 'Long Name',
  className: 'col-auto',
  dataAccessorFn: buildEllipsisTextCell,
  sortFn: (_, order) => sortFn('longName', order)
},
```